stories/css-shadow-variable-clash.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@compiled/react';
+
+export default {
+  title: 'css prop/variable clash',
+};
+
+const isPrimary = true;
```

Comment:
Maybe add a console.log to use the variable

Reply:
Added `console.log()`